src/app/mytools/my-tool/my-tool.component.html

```diff
@@ -62,17 +62,17 @@
                 </mat-toolbar-row>
               </mat-toolbar>
             </div>
-            <p *ngIf="!groupEntriesObject || !groupEntriesObject.length">You have not registered any tools</p>
+            <p *ngIf="!(hasGroupEntriesObject$ | async)">You have not registered any tools</p>
```

Comment:
Does this mean that when you first go to /my-tools, you will see "You have not registered any tools" before your tools end up displaying? Too lazy to try and run your branch....

Same comment/question for my-workflows below.

Reply:
Nope, progress bar loading removes everything on the page